libs/sdk-bindings/makefile

```diff
@@ -121,7 +121,26 @@ bindings-android: kotlin
 	cd bindings-android && ./gradlew assemble
 	mkdir -p ffi/android
 	cp bindings-android/lib/build/outputs/aar/lib-release.aar ffi/android
-		
+
+bindings-kotlin-multiplatform: ios-universal kotlin
+	cp -r ffi/kotlin/jniLibs/ bindings-kotlin-multiplatform/breez-sdk-kmp/src/androidMain/jniLibs/
+	cp -r kmm/* bindings-kotlin-multiplatform/breez-sdk-kmp/src/
+
+	sed -i '' -e "s/\"breez_sdk\"/\"breez_sdk_bindings\"/" bindings-kotlin-multiplatform/breez-sdk-kmp/src/jvmMain/kotlin/breez_sdk/UniFFILib.kt
```

Comment:
@roeierez this is the only thing i couldn't manage to find a solution. I reside to `sed` for text replacement.

Reply:
Here is the line where the binary name is taken: https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings/-/blob/main/src/lib.rs?ref_type=heads#L65

It takes the namespace name as the default.
You can override it by setting it on the config as can be seen here: https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings/-/blob/main/src/lib.rs?ref_type=heads#L45

Reply:
I added the following to `uniffi.toml` and worked.

```
[bindings.kotlin-native]
package_name = "breez_sdk"
cdylib_name = "breez_sdk_bindings"
```

The key is mentioned here: https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings/-/blob/main/src/lib.rs?ref_type=heads#L56